### PR TITLE
Add website dir for reference and add environment/ requirements

### PR DESCRIPTION
**Pull request by Will-Cooper**

### CODE REVIEW

**Comment by dr-rodriguez** [comment]
I think applications that host Flask applications rely on requirements.txt, rather than environment.yml. I worry that this at the top level could cause issues.

**Comment by Will-Cooper** [comment]
Sure, the environment.yml is just for people setting up conda enviros

**Comment by Will-Cooper** [comment]
requirements.txt added

**Comment by Will-Cooper** [comment]
/website files are just to pull in various bokeh functionality

File: environment.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+python-dateutil           2.8.1                      py_0  
+python-utils              2.4.0                    pypi_0    pypi
+pytz                      2020.1                   pypi_0    pypi
+pyyaml                    5.3.1            py38h7b6447c_1  
+qt                        5.9.7                h5867ecd_1  
+readline                  8.0                  h7b6447c_0  
+requests                  2.24.0                   pypi_0    pypi
+scipy                     1.5.0            py38h0b6359f_0  
+secretstorage             3.1.2                    pypi_0    pypi
+sedkit                    1.0.6                    pypi_0    pypi
```
Comment (Will-Cooper) [inline environment.yml:95]: for whatever reason pycharm can't find "import SEDkit", maybe case issue? We can remove anywho

**Comment by Will-Cooper** [comment]
Will keep as draft until we know these environment and requirements work with the website code.
Website subdir will be removed when we've taken what we need, it is just there to draw from

**Pull request closed without merging**